### ISSUE TITLE
Option writeBufferHighWaterMark is defined twice

### ISSUE THREAD

**Issue by timohund** [JanusGraph/janusgraph-docker#11]
The property writeBufferHighWaterMark is defined twice 

once in
https://github.com/JanusGraph/janusgraph-docker/blob/master/example-config/gremlin-server.yaml#L38

and

https://github.com/JanusGraph/janusgraph-docker/blob/master/example-config/gremlin-server.yaml#L39

**Comment by chupman** [comment]
Thanks for taking the time to report the issue. 

I'm going to leave this for now as a good first issue, but it'll need to get fixed before we push out the official images. This also brings attention to the need to setup milestones for this repo.

Instead of 
```
writeBufferHighWaterMark: 32768
writeBufferHighWaterMark: 65536
```
it should be
```
writeBufferLowWaterMark: 32768
writeBufferHighWaterMark: 65536
```

**Comment by timohund** [comment]
@chupman i've created a pr for that

**Comment by ailox** [comment]
I think the related PR was #62. It was merged, so this issue can be closed, no? Am I missing something?

**Comment by FlorianHockmann** [comment]
You're right, @ailox, thanks for noting! Looks like we forgot to link this issue to the PR. I'll close this now.

Fixed in #62.
